scripts/smb-vuln-double-pulsar.nse

```diff
@@ -0,0 +1,140 @@
+local bin = require "bin"
+local smb = require "smb"
+local vulns = require "vulns"
+local stdnse = require "stdnse"
+
+description = [[
+Checks if the target machine is running the Double Pulsar SMB backdoor.
+
```

Comment:
Does this require credentials? Host account creds or special backdoor auth creds? This could affect the CVSS score you calculated.